server/src/main/java/org/apache/druid/curator/discovery/DiscoveryModule.java

```diff
@@ -436,6 +430,12 @@ public void start()
         // nothing
       }
 
+      @Override
+      public CountDownLatch startImmediate()
+      {
+        return null;
```

Comment:
nit: I looked around and don't _think_ anything would be calling this, but maybe it would be safer to return a `CountDownLatch` with `0` as the count just in case anything does?